.github/workflows/build.yml

```diff
@@ -56,5 +55,3 @@ jobs:
     with:
       architecture: ${{ matrix.architecture }}
       distribution: ${{ matrix.distribution }}
-      min: wazuh-indexer-min_${{ needs.version.outputs.version }}-${{ inputs.revision }}-${{ matrix.architecture }}_${{ github.sha }}.${{ matrix.distribution }}
```

Comment:
Parameter `min` is required, and must match the artifact name uploaded by the build job.

`min: ${{ needs.build.outputs.artifact_name }}`